package/MDAnalysis/analysis/hydrogenbonds/hbond_analysis.py

```diff
@@ -206,6 +209,14 @@ def __init__(self, universe, donors_sel=None, hydrogens_sel=None, acceptors_sel=
         acceptors_sel : str
             Selection string for the hydrogen bond acceptor atoms. Leave as `None` to guess which atoms to use in the
             analysis using :attr:`guess_acceptors`
+        between : List (optional)
+            Specify two selection strings for non-updating atom groups between which hydrogen bonds will be calculated.
+            For example, if the donor and acceptor selections include both protein and water, it is possible to
+            find only protein-water hydrogen bonds - and not protein-protein or water-water - by specifying
+            :attr`between=["protein", "SOL"]`. If a two-dimensional list is passed, hydrogen bonds between each pair
+            will be found. For example, :attr`between=[["protein", "SOL"], ["protein", "protein"]]` will calculate
+            all protein-water and protein-protein hydrogen bonds but not water-water hydrogen bonds. If None, hydrogen
```

Comment:
Do you mean `None` (Python value) or string “None”? Use markup.

Is `None` the default?

Reply:
Add a versionadded 2.0.0 for between.